config/env/default.js

```diff
@@ -5,7 +5,8 @@ module.exports = {
     title: 'MEAN.JS',
     description: 'Full-Stack JavaScript with MongoDB, Express, AngularJS, and Node.js',
     keywords: 'mongodb, express, angularjs, node.js, mongoose, passport',
-    googleAnalyticsTrackingID: process.env.GOOGLE_ANALYTICS_TRACKING_ID || 'GOOGLE_ANALYTICS_TRACKING_ID'
+    googleAnalyticsTrackingID: process.env.GOOGLE_ANALYTICS_TRACKING_ID || 'GOOGLE_ANALYTICS_TRACKING_ID',
+    domain: process.env.DOMAIN
```

Comment:
Ah, yes, that would work. But what purpose does it serve? If the domain is missing it is already set dynamically in `express.js`.

Also I would only do this in `development.js`, not in `default.js` because `'localhost' + port` is most certainly wrong for production. Setting it `development.js` would prevent the 'no-domain-set' warning in development mode, so that would be  useful.